src/controller/CommissioningDelegate.h

```diff
@@ -40,6 +40,8 @@ enum CommissioningStage : uint8_t
     kReadCommissioningInfo2,     ///< Query SupportsConcurrentConnection, ICD state, check for matching fabric
     kArmFailsafe,                ///< Send ArmFailSafe (0x30:0) command to the device
     kConfigRegulatory,           ///< Send SetRegulatoryConfig (0x30:2) command to the device
+    kGetTCAcknowledgments,       ///< Waiting for the higher layer to provide terms and conditions acknowledgements.
```

Comment:
might need to move this lower in the list (similar to kScanNetworks and kNeedsNetworkCreds) since those also are optional and might break the cirque tests

Reply:
It looks like it needs to be at the bottom for the python tests.